### DISCUSSION TITLE
Rules for Cookie Values

### DISCUSSION THREAD

**Discussion by gamedevtutorials** [mockoon/mockoon#474] [Ideas] [answered]
Hey. i am using Mockoon a lot for my mocks and was stumbling on the topic to send different Values for different User Sessions.
First of all i provided a Http Only Cookie successfully. Now here comes the point: When i try to create a rule for a cookie value i need to go the way with the Cookie Header which can include multiple Cookies.
So my rules always need to have a regular expression to find a cookie Value.
Would be cool to ask for Cookie Rules
![image](https://user-images.githubusercontent.com/17945179/117485843-2ad59780-af69-11eb-85d2-6bab19b85db5.png)
I tried my best to implement it but i failed ^^
I saw inside the server.ts that mockoon uses the cookieParser so i tried adding it to the response-rule-interceptor like this.
![image](https://user-images.githubusercontent.com/17945179/117486203-a46d8580-af69-11eb-98ca-2dfbab6f74aa.png)

What do you think about this feature? Would be really helpful for me and my colleagues (and other people too ;) )

Best wishes

**Comment by 255kb** [comment, marked as answer]
Hi @gamedevtutorials It's true that cookies are set through the headers, but they have a special treatment, so it would make sense to have a new type of rules to check them. 
Could you please [create an issue](https://github.com/mockoon/mockoon/issues/new?assignees=&labels=&template=Feature_request.md) following the template, so we can track this? Thanks!

**Reply by gamedevtutorials** [reply]
Hi @255kb,
thanks for tracking this problem! I created an issue for this now: https://github.com/mockoon/mockoon/issues/475
Thank you for all the work :)
If you need any further information just tell me.
